manageiq-appliance_console.gemspec

```diff
@@ -25,15 +25,16 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "awesome_spawn",           "~> 1.6"
   spec.add_runtime_dependency "bcrypt",                  "~> 3.1.10"
   spec.add_runtime_dependency "bcrypt_pbkdf",            ">= 1.0", "< 2.0"
+  spec.add_runtime_dependency "ed25519",                 ">= 1.2", "< 1.3"
```

Comment:
Is there a problem with 1.3?

Reply:
Ah 1.3 was what they had in the description of the linked issue https://github.com/net-ssh/net-ssh/issues/565 but the actual documentation does have `< 2.0` I'll update to use that instead.

Reply:
Done